source/features/tag-changes-link.tsx

```diff
@@ -33,8 +33,15 @@ async function getNextPage(): Promise<DocumentFragment> {
 }
 
 function parseTags(element: HTMLElement): TagDetails {
-	// Safari doesn't correctly parse links if they're loaded via AJAX #3899
-	const {pathname: tagUrl} = new URL($(['a[href*="/tree/"]', 'a[href*="/tag/"]'], element)!.href);
+	let tagUrl = $(['a[href*="/tree/"]', 'a[href*="/tag/"]'], element)!.href;
+
+	// If tagUrl from next page, it is just a pathname not a URL
```

Comment:
What does this mean? The `href` property is never relative, it's always the computed full URL; this is not the same as the `href` _attribute_, which _can_ be relative (but we're not using that anywhere)

Reply:
Given that the Safari bug was fixed, I think you just want to use `const tagUrl = $().pathname`